charts/chainlink-cluster/dashboard/dashboard.go

```diff
@@ -351,7 +822,89 @@ func (m *CLClusterDashboard) generate() error {
 			),
 		),
 		dashboard.Row(
-			"Pipeline Tasks Metrics (App)",
+			"Pipeline Metrics (Runner)",
+			row.Collapse(),
+			m.timeseriesRowOption(
+				"Pipeline Task Execution Time",
+				"Sec",
+				`pipeline_task_execution_time{namespace="${namespace}"}`,
+				"{{pod}}",
+			),
+			m.timeseriesRowOption(
+				"Pipeline Run Errors",
+				"",
+				`pipeline_run_errors{namespace="${namespace}"}`,
+				"{{pod}}",
+			),
+			m.timeseriesRowOption(
+				"Pipeline Run Total Time to Completion",
+				"Sec",
+				`pipeline_run_total_time_to_completion{namespace="${namespace}"}`,
+				"{{pod}}",
+			),
+			m.timeseriesRowOption(
+				"Pipeline Tasks Total Finished",
+				"",
+				`pipeline_tasks_total_finished{namespace="${namespace}"}`,
+				"{{pod}}",
+			),
+		),
+		dashboard.Row(
+			"Pipeline Metrics (ETHCall)",
+			row.Collapse(),
+			m.timeseriesRowOption(
+				"Pipeline Task ETH Call Execution Time",
+				"Sec",
+				`pipeline_task_eth_call_execution_time{namespace="${namespace}"}`,
+				"{{pod}}",
+			),
+		),
+		dashboard.Row(
+			"Pipeline Metrics (HTTP)",
+			row.Collapse(),
+			m.timeseriesRowOption(
+				"Pipeline Task HTTP Fetch Time",
+				"Sec",
+				`pipeline_task_http_fetch_time{namespace="${namespace}"}`,
+				"{{pod}}",
+			),
+			m.timeseriesRowOption(
+				"Pipeline Task HTTP Response Body Size",
+				"Sec",
+				`pipeline_task_http_response_body_size{namespace="${namespace}"}`,
+				"{{pod}}",
+			),
+		),
+		dashboard.Row(
+			"Pipeline Metrics (Bridge)",
+			row.Collapse(),
+			m.timeseriesRowOption(
+				"Pipeline Bridge Latency",
+				"Sec",
+				`bridge_latency_seconds{namespace="${namespace}"}`,
+				"{{pod}}",
+			),
+			m.timeseriesRowOption(
+				"Pipeline Bridge Errors Total",
+				"",
+				`bridge_errors_total{namespace="${namespace}"}`,
+				"{{pod}}",
+			),
+			m.timeseriesRowOption(
+				"Pipeline Bridge Cache Hits Total",
+				"",
+				`bridge_cache_hits_total{namespace="${namespace}"}`,
+				"{{pod}}",
+			),
+			m.timeseriesRowOption(
+				"Pipeline Bridge Cache Errors Total",
+				"",
+				`bridge_cache_errors_total{namespace="${namespace}"}`,
+				"{{pod}}",
+			),
+		),
+		dashboard.Row(
```

Comment:
curious how the decision was taken which metrics to include and which to exclude?

Reply:
We include all the core metrics that are not product specific

Reply:
Excluded metrics should be declared/rendered in plugins code